Werehouse/Werehouse.pyproj

```diff
@@ -0,0 +1,35 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">

```

Comment:
general practice is to avoid committing files like `pyproj` which are specific to the IDE so please remove it from the PR

Reply:
deleted! thank U :)